go/vt/worker/legacy_split_clone.go

```diff
@@ -628,9 +633,23 @@ func (scw *LegacySplitCloneWorker) copy(ctx context.Context) error {
 				bls := &binlogdatapb.BinlogSource{
 					Keyspace: src.Keyspace(),
 					Shard:    src.ShardName(),
-					KeyRange: kr,
+					// @bramos: this is meant for a merge so we want to unconditionally replicate
+					// the table to the destination shard
```

Comment:
This comment doesn't make much sense without _just knowing_ that there should be a `KeyRange` attribute here

Reply:
you're totes right -- i'll actually leave the code and comment it out and be a bit clearer.